blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -41,6 +41,8 @@ Unreleased Changes
 Breaking Changes
 ================
 
+- Removed the experimental `enable_semijoin` session setting.
```

Comment:
maybe we should also highlight that, as a consequence, the rewrite/optimization is gone (so its not only about a setting removal).